controls/SV-204440.rb

```diff
@@ -19,48 +18,46 @@
     Enter password:
     Confirm password:'
   impact 0.7
-  tag 'legacy': ['SV-95719', 'V-81007']
-  tag 'severity': 'high'
-  tag 'gtitle': 'SRG-OS-000080-GPOS-00048'
-  tag 'gid': 'V-204440'
-  tag 'rid': 'SV-204440r744098_rule'
-  tag 'stig_id': 'RHEL-07-010491'
-  tag 'fix_id': 'F-4564r744097_fix'
-  tag 'cci': ['CCI-000213']
+  tag legacy: ['SV-95719', 'V-81007']
+  tag severity: 'high'
+  tag gtitle: 'SRG-OS-000080-GPOS-00048'
+  tag gid: 'V-204440'
+  tag rid: 'SV-204440r744098_rule'
+  tag stig_id: 'RHEL-07-010491'
+  tag fix_id: 'F-4564r744097_fix'
+  tag cci: ['CCI-000213']
   tag nist: ['AC-3']
-  tag subsystems: ["boot", "uefi"]
+  tag subsystems: ['boot', 'uefi']
   tag 'host'
 
   if virtualization.system.eql?('docker')
```

Comment:
similar to 204438, we might be skipping out via the bios claim incorrectly.  https://docs.aws.amazon.com/AWSEC2/latest/UserGuide/virtualization_types.html.  but then again the management console page for the ec2 instances says both of them were `Current instance boot mode: legacy-bios` so idk lol.  i don't know enough about how aws does stuff.

Reply:
Historically, AWS support has been a separate effort because we have to consider a lot of different ways of testing the same STIG rule (for SV-204438 and SV-204440, it could be checking parameter values, AWS EC2-specific file paths, etc). For now, I've updated @em-c-rod's [issue](https://github.com/mitre/redhat-enterprise-linux-7-stig-baseline/issues/206) to include this AWS support suggestion. Thanks @Amndeep7